src/test/java/org/gridsuite/modification/server/modifications/LoadCreationInBusBreakerTest.java

```diff
@@ -44,6 +46,7 @@ protected ModificationInfos buildModification() {
             .reactivePower(30.0)
             .connectionName("top")
             .connectionDirection(ConnectablePosition.Direction.TOP)
+            .properties(List.of(FreePropertyInfos.builder().name("property-name").value("property-value").build()))
```

Comment:
"property-name" and "property-value" are used may times. Shouldn't they be centralised static Strings (DEFAULT_PROPERTY_NAME or whatever, given that they seem to be tested as such) ?